pinlockview/src/main/java/com/andrognito/pinlockview/PinLockView.java

```diff
@@ -165,6 +199,15 @@ private void init(AttributeSet attributeSet, int defStyle) {
             mDeprecatedColorOptions = typedArray.getBoolean(R.styleable.PinLockView_keypadUseDeprecatedColorOptions, true);
             mTextColor = typedArray.getColor(R.styleable.PinLockView_keypadTextColor, ResourceUtils.getColor(getContext(), R.color.white));
             mTextSize = (int) typedArray.getDimension(R.styleable.PinLockView_keypadTextSize, ResourceUtils.getDimensionInPx(getContext(), R.dimen.default_text_size));
+            mShowEnterButton = typedArray.getBoolean(R.styleable.PinLockView_keypadShowEnterButton, false);
+            if (mShowEnterButton) {
+                mShowDeleteButton = true;
```

Comment:
why is show delete button not its own setting?

Reply:
It is it's own setting (Line 193). Looks like this is a leftover from the fork I derived this from and I forgot to remove it.